package.json

```diff
@@ -35,13 +30,13 @@
   },
   "dependencies": {
     "@apollo/react-common": "^3.1.4",
-    "@apollo/react-components": "^3.1.5",
-    "@apollo/react-hoc": "^3.1.5",
-    "@apollo/react-hooks": "^3.1.5",
-    "@jahia/data-helper": "1.0.12",
+    "@apollo/react-components": "^4.0.0",
+    "@apollo/react-hoc": "^4.0.0",
+    "@apollo/react-hooks": "^4.0.0",
```

Comment:
It's probably better to move to official @apollo/client package as this project is deprecated https://github.com/apollographql/react-apollo.